storages/chat-store/migrations/2026-07-24-000000_bare_identity_keys/up.sql

```
@@ -0,0 +1,64 @@
+            WHEN instr(message_receipts.user_jid, ':') > 0
+            THEN substr(message_receipts.user_jid, 1, instr(message_receipts.user_jid, ':') - 1)
+                 || substr(message_receipts.user_jid, instr(message_receipts.user_jid, '@'))
+            ELSE message_receipts.user_jid
+          END);
+
+UPDATE OR IGNORE message_receipts
+   SET user_jid = substr(user_jid, 1, instr(user_jid, ':') - 1)
+                  || substr(user_jid, instr(user_jid, '@'))
+ WHERE user_jid LIKE '%:%@%';
+
+DELETE FROM message_receipts WHERE user_jid LIKE '%:%@%';
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)